lib/logstash/inputs/udp.rb

```diff
@@ -50,6 +51,7 @@ def run(output_queue)
       udp_listener(output_queue)
     rescue => e
       @logger.warn("UDP listener died", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
I will apply here same pattern as in https://github.com/logstash-plugins/logstash-input-syslog/pull/28